docs/tutorial.md

```diff
@@ -556,3 +558,29 @@ status:
 
 The status of type `Succeeded = True` shows the pipeline ran successfully, also
 the status of individual Task runs are shown.
+
+
+## Local development
+
+### Known good configuration
+
+Knative (as of version 0.3) is known to work with:
+- [Docker for Desktop](https://www.docker.com/products/docker-desktop): a version that uses Kubernetes 1.11 or higher. At the time of this document, this requires the *edge* version of Docker to be installed
+- The following [prerequisites](https://github.com/knative/build-pipeline/blob/master/DEVELOPMENT.md#requirements)
+- Setting `host.docker.local:5000` as an insecure registry with Docker for Desktop
```

Comment:
do you happen to have a link to docs on how to set this as an insecure registry?

Reply:
There's a lil whale we can click under Docker for Desktop -> Daemon, the documentation [here](https://docs.docker.com/registry/insecure/) isn't the most concise or I'd gladly refer to this. I'll mention it's under the preferences - I also need to shout out that running knative and istio requires *lots* of CPU and memory!